### PR TITLE
HCAP-1024 | Refactor fetch participants

### PR DESCRIPTION
**Pull request by arranfw**
https://freshworks.atlassian.net/browse/HCAP-1024

### CODE REVIEW

File: client/src/pages/private/ParticipantTable.js [hunk 1]
```diff
@@ -233,46 +191,10 @@ const ParticipantTable = () => {
...
-    filter,
-    order,
-    roles,
-    columns,
-    selectedTab,
-    selectedTabStatuses,
-    participantsDispatch,
-  ]);
+    fetchParticipants();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (dbayly-freshworks) [inline client/src/pages/private/ParticipantTable.js:196]: I gather the linter is complaining here, so this feels like a bit of a smell. Have you looked into resolving this hook dependency issue?

**Review by dbayly-freshworks — APPROVED** [review]
One minor concern but I like the direction this is going

**Review by fwpushan — DISMISSED** [review]
LGTM

**Review by arranfw — COMMENTED** [review]
Ugh I forgot to post this reply

**Review by dbayly-freshworks — APPROVED** [review]

**Pull request merged**